engine/src/main/java/com/sandboni/core/engine/Processor.java

```diff
@@ -107,8 +109,17 @@ private Builder getBuilder() {
     private boolean proceed(ChangeScope<Change> changeScope) {
```

Comment:
Delete the JavaDoc comment from this method, is outdated and hard to update. Let the code explains by itself.